.github/workflows/release-snapshot.yaml

```diff
@@ -0,0 +1,54 @@
+name: Release SHAPSHOT
+
+on:
+  push:
+    branches:
+      - master
+      - "[0-9]+"
+      - "[0-9]+.[0-9]+"
```

Comment:
This workflow targets the `master` branch, the support branches (like `3`), and the release branches (like `3.4` and `3.5`). The triggers are `push` to those branches.